portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,46 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+let curSlide = 0; //By default, display first slide.
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/*
+ * Advance forward by one slide.
+ */
+function forward() {
+  updateSlide(curSlide + 1);
+}
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/*
+ * Advance backwards by one slide.
+ */
+function back() {
+  updateSlide(curSlide - 1);
 }
+
+/*
+ * Display slide, and hide all other slide elements.
+ */
+function updateSlide(slide) {
+  let slideArr = document.getElementsByClassName("slide");
+  let totalSlides = slideArr.length;
+
+  //Adjust curSlide  value.
+  if (slide >= totalSlides) {
+    slide = 0;
+  } else if (slide < 0) {
+    slide = totalSlides - 1;
+  }
+  curSlide = slide;
+
+  let defaultSlide = document.getElementsByClassName("default-slide");
+  if (defaultSlide.length != 1) return;
+  defaultSlide[0].style.display = "none";
```

Comment:
Instead of modifying the style here, it might be better to add/remove a class that displays/hides the slide.  You can use classList.add and classList.remove:
https://www.w3schools.com/jsref/prop_element_classlist.asp

Reply:
My current definition of default-slide and slide include multiple properties, so I ended up creating a class with just the line `display: none;`, and adding/removing that from slides in script.jss